### ISSUE TITLE
using the code

### ISSUE THREAD

**Issue by MayAndi97** [dsjh17/Autoencoder-for-Anomaly-Detection-on-MetroPT-dataset#1]
I would like to build on this code in my bachelor's thesis, would that be okay with you? I would also be happy to quote you, including a link to the repository. Greetings MayAndi97

**Comment by dsjh17** [comment]
I am glad that my code is useful and you are welcome to build on it. Also, would you be willing to share the topic of your thesis? It may serve as inspiration for my own thesis in the future as I am currently studying data science at a university.

**Comment by MayAndi97** [comment]
Of course, the work involves applying methods from statistical process control to the output of the autoencoder instead of setting a hard threshold.

**Comment by dsjh17** [comment]
Thank you for sharing your approach and all the best for your thesis.
